generator/dart/packages/generated/rpc/types/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Code generated by sidekick. DO NOT EDIT.
+
+name: google-cloud-rpc
```

Comment:
nit: these names need to be valid dart identifiers; so `google-cloud-rpc` => `google_cloud_rpc`.

Reply:
Done. The source of truth for this is the `.sidekick.toml` file, so I fixed it there and regenerated.